### ISSUE TITLE
this is still possible, my binds are still changed.

### ISSUE THREAD

**Issue by All-github-usernames-taken** [ValveSoftware/halflife#1898]
https://old.reddit.com/r/counterstrike/comments/97opg6/can_i_get_a_virus_from_16/e4buqmk/

**Comment by djdallmann** [comment]
Yes it's possible, don't join shady or lesser known servers and you should be ok.

**Comment by All-github-usernames-taken** [comment]
What the fuck are you on about, how are you supposed to know if a server is shady or not?

It's obvious to me, but I guess we should just expect the user to know now, and its their problem if they get that sort of mean virus on their PC? Fuck me.

What if I told you, that servers can execute commands on you instantly after and while you load into their servers? 

Do you have any idea what you are talking about here?

**Comment by djdallmann** [comment]
I agree it's unfortunate that this could happen but you can't know 100% if a server is safe or will abuse it's power, it's just like installing apps on your phone/computer/tablet, you can't trust that every app is safe and won't steal information etc. Also this is a 20 year old game and barely maintained. Valve has put in some protections against this however people always find ways around protections.

- Make sure your PC Operating system (not winxp, it's not supported anymore) and software (Java/Flash/Adobe/browsers) are up to date
- Play on servers that have been around a while or have a decent size community
- Don't install pirated software
- Use an Anti-Malware or Anti-Virus product
